Examples/hello-world/Protos.proto

```diff
@@ -1,4 +1,4 @@
-// Copyright 2015 gRPC authors.
```

Comment:
Not sure what's happened here, but this is meant to be a symlink from `Examples/hello-world/Protos/HelloWorld.proto` to `dev/protos/upstream/grpc/examples/helloworld.proto`. Could you fix it?